src/components/Select.jsx

```diff
@@ -217,19 +217,17 @@ export default class Select extends Component {
     let focusedOptionValue;
 
     if (has(properties, 'valueLink')) {
-      selectedValue = properties.valueLink.value;
-      focusedOptionValue = selectedValue;
+      focusedOptionValue = selectedValue = properties.valueLink.value;
```

Comment:
Multiple left-hand assignments have the strange side-effect in JS that they are then in the global scope. From what I learned it's best to avoid them at all. see http://stackoverflow.com/a/1758912/837709